### ISSUE TITLE
Pyre's Cherenkov Drive and Argent Adept's instruments

### ISSUE THREAD

**Issue by jkittycat** [SotMSteamMods/CauldronMods#980]
**Describe the bug**
Attempting to use the effect of Pyre's Cherenkov Drive on Argent Adept's instrument cards fails.

**What does it affect**
Card(s): Cherenkov Drive, Argent Adept's instrument cards
Deck(s): Pyre, Argent Adept

**To Reproduce**
1. Play Cherenkov Drive
2. Use Cherenkov Drive's effect on an instrument in Argent Adept's hand

**Expected behavior**
To be able to use the effects of melody/harmony/rhythm cards in play.

**Screenshots**
![20210223162637_1](https://user-images.githubusercontent.com/77370875/108918479-0ec38a80-75f7-11eb-9c70-283813d456a7.jpg)

**Logs**

**Additional context**
I'm guessing there's built in checks if AA's instruments are in play because of Cedistic Dissonant.

**Comment by LNBertolini** [comment]
Yep. Not much we can do about this ourselves, unless we do something really weird with sneaking the card into play long enough to use the power without alerting any triggers. Probably best not to try.
